src/Robo/Commands/Blt/DevCommand.php

```diff
@@ -13,10 +13,14 @@ class DevCommand extends BltTasks {
   /**
    * Links to a local BLT package via a Composer path repository.
    *
+   * Also sets up DrupalVM NFS mounts to use this BLT path. This currently
+   * requires the BLT path to be '../../packages/blt'.
+   *
    * @command blt:dev:link-composer
+   *
    * @param array $options
    */
-  public function linkComposer($options = ['blt-path' => InputOption::VALUE_REQUIRED]) {
+  public function linkComposer($options = ['blt-path' => '../../packages/blt']) {
     $composer_json_filepath = $this->getConfigValue('repo.root') . '/composer.json';
     $composer_json = json_decode(file_get_contents($composer_json_filepath));
     $composer_json->repositories->blt = [
```

Comment:
I just realized that instead of muddling with json here, I think I can just shell out to `composer config repositories.blt path $options['blt-path']`